backend/src/test_home.rs

```diff
@@ -0,0 +1,12 @@
+use super::rocket;
+use rocket::http::Status;
+use rocket::local::blocking::Client;
+
+// Just a default test to make sure CI is running properly
+#[test]
+fn home() {
+    let client = Client::tracked(rocket()).expect("valid rocket instance");
+    let response = client.get(uri!(super::home)).dispatch();
+    assert_eq!(response.status(), Status::Ok);
+    assert_eq!(response.into_string().unwrap(), "Hello, world!");
+}
```

Comment:
Just a note - if we are testing simple service layer functions or smaller components, we would put these into unit testing. If we are testing API endpoints, this will be integration testing

Reply:
understood, in the [rust book](https://doc.rust-lang.org/cargo/guide/tests.html#tests) there was a guideline were unit tests just dwell inside the src folder, and integration tests go into the tests folder